src/index.js

```diff
@@ -1,65 +1,16 @@
-import firebase from 'firebase/compat/app';
-import * as firebaseui from 'firebaseui'
-import 'firebaseui/dist/firebaseui.css'
+import Vue from "vue";
```

Comment:
Stripped out all the firebase config and sent that to firebase.js

Import all the Vue good stuff from the npm modules